modules/react/table/stories/stories_visualTesting.tsx

```diff
@@ -66,66 +67,45 @@ const exampleData = [
   },
 ];
 
+const tableHeaderStencil = createStencil({
+  base: {
+    backgroundColor: system.color.bg.alt.softer,
+    borderRight: `1px solid ${system.color.border.divider}`,
+  },
+  modifiers: {
+    variant: {
+      sticky: {
+        position: 'sticky',
+        left: system.space.zero,
+      },
+    },
+  },
+});
+
 const FixedColumn = () => (
   <>
     <Heading size="small">Table Heading</Heading>
     <Table>
       <Table.Head>
         <Table.Row>
-          <Table.Header
-            position="sticky"
-            left="0"
-            backgroundColor="soap100"
-            borderRight={`1px solid ${colors.soap400}`}
-          >
-            Header
-          </Table.Header>
-          <Table.Header borderRight={`1px solid ${colors.soap400}`} backgroundColor="soap100">
-            Header
-          </Table.Header>
-          <Table.Header borderRight={`1px solid ${colors.soap400}`} backgroundColor="soap100">
-            Header
-          </Table.Header>
-          <Table.Header borderRight={`1px solid ${colors.soap400}`} backgroundColor="soap100">
-            Header
-          </Table.Header>
-          <Table.Header borderRight={`1px solid ${colors.soap400}`} backgroundColor="soap100">
-            Header
-          </Table.Header>
-          <Table.Header borderRight={`1px solid ${colors.soap400}`} backgroundColor="soap100">
-            Header
-          </Table.Header>
-          <Table.Header borderRight={`1px solid ${colors.soap400}`} backgroundColor="soap100">
-            Header
-          </Table.Header>
-          <Table.Header borderRight={`1px solid ${colors.soap400}`} backgroundColor="soap100">
-            Header
-          </Table.Header>
-          <Table.Header borderRight={`1px solid ${colors.soap400}`} backgroundColor="soap100">
+          <Table.Header {...mergeStyles(tableHeaderStencil({variant: 'sticky'}))}>
```

Comment:
you don't need `mergeStyles`, it should be used only for elements that having style props passed, as `mergeStyles(elemProps, tableHeaderStencil({...})`.

you should be able to spread it as

```tsx
<Table.Header {...tableHeaderStencil({variant: 'sticky'})}>
```

Reply:
That's right. Thanks for pointing that out!